example_test.go

```diff
@@ -10,10 +12,10 @@ import (
 	"log"
 	"os"
 	"os/signal"
+	"syscall"
 	"time"
 
 	"github.com/hibiken/asynq"
-	"golang.org/x/sys/unix"
```

Comment:
syscall.SIG* have been replaced with unix.SIG* in signals_unix.go.

syscall.SIG* did not replaced with windows.SIG* in signals_windows.go because QUIT with CTRL-C not supported in Windows if signal notify windows.SIGTERM.